docs/utilities/tags.md

```diff
@@ -9,13 +9,22 @@ Tags are declared in your mod's [datapack][datapack]. For example, `/data/<modid
 Similarly, you may append to or override tags declared in other domains, such as Vanilla, by declaring your own JSONs.
 For example, to add your own mod's saplings to the Vanilla sapling tag, you would specify it in `/data/minecraft/tags/blocks/saplings.json`, and Vanilla will merge everything into one tag at reload, if the `replace` option is false.
 If `replace` is true, then all entries before the json specifying `replace` will be removed.
-See the [Vanilla wiki][tags] for a description of the base syntax.
+Values listed that are not present will cause the tag to error unless the value is listed using an `id` string and `required` boolean set to false as in the following example.
 
-Forge provides two extensions on the Vanilla syntax:
+As an example:
+```json
+{ "replace": false,
+  "values": [
+    "minecraft:gold_ingot",
+    "mymod:my_ingot",
+    { "id": "othermod:ingot_other", "required": false }
+  ]
+}
```

Comment:
JSON should be formatted in pretty printing since that's how json objects are generated.

Example:
```json
{
  "key1": "value1",
  "key2": [
    "list1",
    "list2",
    {
      "innerkey1": "innervalue1",
      "innerkey2": "innervalue2"
    }
  ]
}
```